reference_guide/ui_themes/themes_customize.md

```diff
@@ -166,6 +178,49 @@ The following example sets the background color for all labels to the color #F6E
 ```
 The `Label.background` entry supersedes, in the narrower context of label backgrounds, any default color as well as any wildcard color assigned to backgrounds.
 
+#### Customizing the Color of UI Tabs
+The color of UI Tabs are changed by adding a key-value pair to the `"ui": {}` section of a Theme description file.
+The `key` is the `element.property` format and the `value` is the custom color.
```

Comment:
How to customize control colors is described under the header "Customizing the Color of Specific UI Control Types". Why is it useful to repeat the same description here?

Reply:
If someone jumps to this section I want to be sure they understand "key", "element", and "value". But you are right, repeating the content is unnecessary. I will refer to the previous section instead.